Screens/Profile.tsx

```diff
@@ -0,0 +1,293 @@
+import { StatusBar } from "expo-status-bar";
+import React, { useState } from "react";
+import {
+    Image,
+    ImageBackground,
+    Modal,
+    StyleSheet,
+    Text,
+    TextInput,
+    TouchableOpacity,
+    View,
+} from "react-native";
+import Seperator from "../Components/Seperator";
+import ProfileField from "../Components/ProfileField";
+import { MaterialCommunityIcons } from "@expo/vector-icons";
+import { Formik } from "formik";
+import * as Yup from "yup";
+import CustomButton from "../Components/CustomButton";
+
+type ProfileFormValues = {
+    name: string;
+    email: string;
+    phone: string;
+};
+
+export default function Profile() {
+    const [modalVisible, setModalVisible] = useState(false);
+
+    const initalFormValues: ProfileFormValues = {
+        name: "Cammy White",
+        email: "123@CammyWhite.com",
+        phone: "555-555-555",
+    };
+
+    const ProfileSchema = Yup.object().shape({
+        name: Yup.string()
+            .min(4, "Must be at least 4 characters")
+            .max(50, "Must be less than 50 characters")
+            .required("name is required"),
+        email: Yup.string()
+            .matches(
```

Comment:
it looks like yup has built in email validation and you don't need to write your own regex
you can use this
` .email("Invalid email address")`
Also I think emails can be a maximum of 256 chars so you may want to add something like this
` .max(256, "Must be less than 256 characters")`

Reply:
it does not work